chore: even more doc work (#10)
time-loop/clickup-projen: README.md

When creating new cdk apps:

```bash
mkdir my-new-cdk-app
cd my-new-cdk-app
projen new --from @time-loop/clickup-projen clickupcdk_clickupcdktypescriptapp
gh repo create time-loop/my-new-cdk-app --private --push --source=.
```

### ClickUpTypeScriptProject

When creating new TypeScript Libraries:

```bash

> mkdir my-new-lib
> cd my-new-lib

Reviewer: ```suggestion
mkdir my-new-cdk-app
cd my-new-cdk-app
```

Author: Nope. This is explicitly an example of how to create a TypeScript Project. There's an example of a CdkTypeScriptApp above.

- Standard prettier and eslint configs.
- For CDK apps, CDKv2 support with `@time-loop/cdk-library`, `cdk-constants`, `cdk-iam-floyd`, `colors` and `multi-convention-namer` libraries pre-installed for your coding pleasure.

## What's Left For You To Do

> - Create the repo in GitHub. `gh repo create time-loop/my-new-lib --private --push --source=.` will probably do what you need. Note: the names MUST MATCH.

Reviewer: ```suggestion
- Create the repo in GitHub. `gh repo create time-loop/my-new-cdk-app --private --push --source=.` will probably do what you need. Note: the names MUST MATCH.
```

Reviewer: you get the idea. Do this everywhere.

Author: No. While we have support in here for cdk-apps, it's not my primary focus. We'll probably pound out a few dozen cdk apps. `clickup_frontend` alone has over 300 libraries. So... that's my target audience here.

        - Include administrators ()

That's it! Now go write a failing test and some code to make it pass!

## Use Your New Private Library

### Setup GitHub Packages Access

This will tell `npm` / `yarn` / `pnpm` to find `@time-loop/*` libraries
in the GitHub package registry.

Start by creating a [PAT](https://github.com/settings/tokens) in GitHub
(Settings => Developer Settings => Personal access tokens).
It needs to have at least the `read:packages` permission.
You might want to also use this token with the
[gh](https://github.com/cli/cli) command line tool. Up to you.

Add the following to your `.npmrc`:

```bash
cat <<EOF >>~/.npmrc

//npm.pkg.github.com/:_authToken=ghp_your-github-token
@time-loop:registry=https://npm.pkg.github.com/
EOF
```

> NOTE: There are also some `@clickup/*` libraries in ye olde `npmjs.com`.

Reviewer: 🤠

### Add It To Your App

Now that the GitHub package repo is configured, you can just

```

> yarn add @time-loop/my-new-lib --save

Reviewer: ```suggestion
yarn add @time-loop/my-new-cdk-app --save
```

Author: Nope. I don't have an example of a cdk lib (or, for that matter, support for them outside of `cdk-library`). We might want to create some open-source constructs in the future which would drive this, but for now, we don't have this.
